### PR TITLE
Remove unused messages from delete commands

### LINKED ISSUE #95: Remove unused messages from delete commands
`DeleteExpenseCommand` and `DeleteIncomeCommand` contain message strings not used by the application. They should be removed.

In addition, the messages in `DeleteCommand` are to be expanded to include descriptions of different behavior across tabs.

### PR DESCRIPTION
**Pull request by wltan**
Fixes #95

### CODE REVIEW

**Review by zhaojj2209 — DISMISSED** [review]
LGTM!

File: src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/DeleteExpenseCommand.java [hunk 1]
```diff
@@ -14,15 +14,6 @@
...
 public class DeleteExpenseCommand extends DeleteCommand {
 
-    public static final String COMMAND_WORD = "delete";
-
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the expense identified by the index number used in the displayed expense list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
-
-    public static final String MESSAGE_DELETE_TRANSACTION_SUCCESS = "Deleted Expense: %1$s";
```
Comment (ianyong) [inline src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/DeleteExpenseCommand.java:24]: Currently, these specific success messages are being shown. Should these be removed?

**Review by ianyong — CHANGES_REQUESTED** [review]

File: src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/DeleteExpenseCommand.java (hunk 1 above)
Comment (wltan) [reply src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/DeleteExpenseCommand.java:24]: Fixed

**Review by ianyong — APPROVED** [review]
LGTM!

**Review by zhaojj2209 — APPROVED** [review]
LGTM!

**Pull request merged**